{
  "Header": {
    "Original": "CVE-2023-46445 (An issue in AsyncSSH before 2.14.1 allows attackers to control the ext ...)",
    "ID": "CVE-2023-46445",
    "Description": "(An issue in AsyncSSH before 2.14.1 allows attackers to control the ext ...)"
  },
  "Annotations": [
    {
      "Original": "- python-asyncssh \u003cunfixed\u003e (bug #1056000)",
      "Type": "package",
      "Package": "python-asyncssh",
      "Kind": "unfixed",
      "BugNo": 1056000
    },
    {
      "Original": "[bookworm] - python-asyncssh \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bookworm",
      "Package": "python-asyncssh",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[bullseye] - python-asyncssh \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bullseye",
      "Package": "python-asyncssh",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[buster] - python-asyncssh \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "buster",
      "Package": "python-asyncssh",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/ronf/asyncssh/security/advisories/GHSA-cfc2-wr2v-gxm5",
      "Type": "NOTE",
      "Description": "https://github.com/ronf/asyncssh/security/advisories/GHSA-cfc2-wr2v-gxm5"
    }
  ]
}